glib/src/value.rs

```diff
@@ -246,6 +246,14 @@ where
 }
 
 /// Trait to convert a value to a `Value`.
+///
+/// Similar to other common conversion traits, the following invariants are guaranteed:
+/// 
+/// - **Invertibility**: `x.to_value().get().unwrap() == x`. In words, `FromValue` is the inverse of `ToValue`.
+/// - **Idempotence**: `x.to_value() == x.to_value().to_value()`.
+///   In words, applying `ToValue` multiple times yields the same result as applying it once.
+///   Idempotence also applies the other way around: `value.get::<Value>()` is a no-op.
```

Comment:
That last part is not actually implemented here, is it?

Reply:
Yes it is. The tests do `.to_value().get::<Value>().unwrap().get().unwrap()`.

Reply:
Hm, where does the `FromValue` impl for `Value` come from?

Reply:
Oh!